<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupChatDialog</class>
 <widget class="QDialog" name="GroupChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="te_chat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_send">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="tb_chat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pb_to1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>表情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_to2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_to3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_GroupMer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>271</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>191</width>
     <height>271</height>
    </size>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>179</width>
      <height>269</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QWidget" name="wdg_Merlist" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacerMer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>340</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
